apps/examples/testcase/le_tc/kernel/tc_timer.c

```diff
@@ -44,15 +44,37 @@ static void tc_timer_timer_create_delete(void)
 	int ret_chk = 0;
 	timer_t timer_id;
 	timer_t gtimer_id;
-	clockid_t clockid = CLOCK_REALTIME;
+	clockid_t clockid = CLOCK_MONOTONIC;
```

Comment:
CLOCK_MONOTONIC is defined in time.h and it has ifdef condition. Please check this.